pkg/controller/knativeserving/common/extensions.go

```diff
@@ -5,13 +5,14 @@ import (
 	servingv1alpha1 "github.com/openshift-knative/knative-serving-operator/pkg/apis/serving/v1alpha1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
 var log = logf.Log.WithName("common")
 
-type Platforms []func(client.Client, *runtime.Scheme, *mf.Manifest) (*Extension, error)
+type Platforms []func(client.Client, *runtime.Scheme, *mf.Manifest, *rest.Config) (*Extension, error)
```

Comment:
Is all the config passing still necessary?

Reply:
No its not required now :)